sources/packages/web/src/views/institution/WithdrawalUpload.vue

```diff
@@ -55,7 +55,7 @@
             :clearable="true"
             :accept="ACCEPTED_FILE_TYPE"
             density="compact"
-            v-model="withdrawalFiles"
+            :v-model="withdrawalFiles"
```

Comment:
Did the ":" add any difference? I did not see anywhere else where it was required or in Vue docs.
If it was needed, can you please elaborate on the why?

Reply:
Nope. It was a mistake. I reverted it back.